src/content/desci/index.md

```diff
@@ -8,77 +8,267 @@ sidebar: true
+
+
+## How DeSci improves science
+
+**List of key problems in science and how decentralized science can help to address these issues**
+
+
+
+* bureaucratic funding structures -> allow for more diverse funding sources (quadratic donations, DAOs, crowdfunding etc.) 
+* [replication crisis](https://forrt.org/glossary/reproducibility-crisis-aka-replicab/) -> allowing for easier access to data and methods, and by providing incentives for reproducibility.
+* misaligned incentives -> aligning incentives more closely with the goals of science. 
+* poor data quality -> incentivising transparent data sharing and quality control. 
+* [publication bias](https://forrt.org/glossary/publication-bias-file-drawer-proble/) -> enabling publication in a wider range of outlets. 
+* [p-hacking](https://www.methodspace.com/blog/primer-p-hacking) -> incentivising transparent and honest reporting of results.
+* [cherry picking](https://pubmed.ncbi.nlm.nih.gov/33999541/) -> requiring publication of all results, including negative or inconclusive ones. 
+* [HARKing](https://forrt.org/glossary/harking/): hypothesizing after the results are known -> providing incentives for transparent and honest reporting of results.
+
+A comparison to traditional science
+
+
+<table>
+  <tr>
+   <td><strong>DeSci</strong>
+   </td>
+   <td><strong>Traditional science</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>You raise funding from a decentralized community.
+   </td>
+   <td>You raise funding from a centralized entity.
+   </td>
+  </tr>
+  <tr>
+   <td>You collaborate with peers from all over the globe in dynamic teams.
+   </td>
+   <td>Your collaborations are limited by funding organizations and home institutions.
+   </td>
+  </tr>
+  <tr>
+   <td>Funding decisions are made online and in a transparent way. New funding mechanisms are explored.
+   </td>
+   <td>Funding decisions are made with a long turnaround time and limited transparency. Few funding mechanisms exist.
+   </td>
+  </tr>
+  <tr>
+   <td>DeSci is open to anyone.
+   </td>
+   <td>You must be a member or a physical research institute to do science.
+   </td>
+  </tr>
+  <tr>
+   <td>You can trade laboratory services freely.
+   </td>
+   <td>You are constrained to using your local institute's laboratory infrastructure.
+   </td>
+  </tr>
+  <tr>
+   <td>You publish your work online on your own terms.
+   </td>
+   <td>You publish through established pathways.
+   </td>
+  </tr>
+  <tr>
+   <td>You can earn tokens and reputation for peer-reviewing work.
+   </td>
+   <td>Your peer-review work is unpaid, benefiting for-profit publishers.
+   </td>
+  </tr>
+  <tr>
+   <td>You own the IP you generate and distribute it according to transparent terms.
+   </td>
+   <td>Your home institution owns the IP you generate. Access to the IP is not transparent.
+   </td>
+  </tr>
+</table>
 
-## Why do we need DeSci?
```

Comment:
Proposed Change:

# Why Web3 needs DeSci
## Adoption
Decentralized science has the potential to attract new participants and contributors to web3 and to help disseminate knowledge more effectively.

The web3 community is still in its early stages, and there is a lot of work to be done in order to realize the full potential of decentralized science. However, the potential benefits are significant, and it is worth exploring how decentralized science can be used to improve the way we conduct research and share knowledge.


Participants in Web3
The tools DeSci will produce are easily understood and explained without technical jargon. For example, research objects are scientific papers that contain more information than a simple pdf. Imagine interacting with a research paper instead of reading it. Imagine seeing the code used to run simulations and running it yourself with a simple click of a button. Imagine seeing every change, evolution, and contributor to the research over its lifetime just like with Git. 

Publishing networks, as another example, allow research to be published, reviewed, elevated, and replicated without input from a controlling authority or company. Imagine never paying to access a research object. Imagine researchers getting paid directly for their work instead of through an intermediary. Imagine a marketplace of people and entities that provide review, curation, and replication services instead of an oligopoly of a few major conglomerates. 

## Contributors to Web3
The people and institutions onboarded through DeSci tooling have spent their careers and lifetimes exploring, creating, and translating knowledge from all disciplines. They often work on problems similar to those currently facing the trustless environment. Web3 needs this seemingly limitless experience to help shape the foundation of the macro-systems we are developing. 

DeSci is building the tools that will make their lives easier. In doing so, we invite them into the trustless world to continue doing what they do best: solve the most challenging and fulfilling problems society has to offer.

Reply:
I suggest removing the entire "Why Web3 needs DeSci" section. 

First, if Web3 "needs" DeSci then we have a very big problem. The infrastructure can't rely on one of its applications - it must be the other way around.

Second. "adoption" isn't a unique benefit from DeSci - you could apply this logic to any application.

Third, I don't follow the logic in the "contributors to Web3" section - it suggests that web3 will benefit by having DeSci-pilled scientists working on the stack, but the stack exists to help those same people work on their science, not building out the stack.

I have already explained my concerns around the "legitimacy" section.

I think it is better to remove the section altogether.